examples/abitest/config/config.textproto

```diff
@@ -26,17 +26,27 @@ node_configs {
     address: "test.invalid:9999"
   }
 }
+node_configs {
```

Comment:
Is it possible to update [`oak::DefaultConfig()`](https://github.com/project-oak/oak/blob/master/oak/common/app_config.cc#L44-L56) to include something like this stanza, if it's now effectively required?  Or is that not possible because of the need to know a TLS auth data in advance?

Reply:
This is going away soon anyways.